.github/workflows/meson.yml

```diff
@@ -19,7 +19,8 @@ jobs:
             name: Windows MSVC Release,
             os: windows-latest,
             msvc: true,
-            pkgmanager: choco,
+            pkgmanager: msys,
+            msystem: MINGW64,
             shell: 'powershell',
             buildtype: release,
             args: '-Ddefault_library=static --wrap-mode=forcefallback'
```

Comment:
`--wrap-mode=forcefalback` is now entirely obsolete; though since it still compiles only libass atm i guess this already became obsolete when explicit wrap-fallbacks got removed from our `meson.build`